TempleEight/docs/guide/business-logic.md

```diff
@@ -0,0 +1,93 @@
+---
+id: business-logic
+title: Adding business logic to your service 
+sidebar_label: Business Logic 
+---
+
+We know that the code generated by Temple won't necessarily satisfy every use case.
+That's why we made it easy to modify and extend the application logic, without sacrificing the ability to regenerate code after.
+This short guide will walk you through how to add custom business logic to your application, as well as additional endpoints.
+We'll be using the `ExampleProject` from the [getting started guide](getting-started.md).
```

Comment:
Might be worth saying that this tutorial assumes some level of familiarity with writing Go